src/components/WizardContainer/index.js

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useContext } from 'react';
+import { withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Wizard, Steps, Step } from 'react-albus';
+import Step1 from '../Step1';
+import Step2 from '../Step2';
+import WizardContext from '../../context/WizardContext';
+
+import styles from './index.module.scss';
+
+const stepContainers = [
+  {
+    id: 'step1',
+    content: Step1,
+  },
+  {
+    id: 'step2',
+    content: Step2,
+  },
+];
+
+const WizardContainer = (props) => {
```

Comment:
Just call this component Wizard, you can avoid conflicts by doing

`import { Wizard as AlbusWizard, Steps, Step } from 'react-albus';`